GitUI/Properties/Images.resx

```diff
@@ -691,6 +691,9 @@
   <data name="WorkTree" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>..\Resources\Icons\WorkTree.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
+  <data name="WorkTreeDeleted" type="System.Resources.ResXFileRef, System.Windows.Forms">
+    <value>..\Resources\Icons\WorkTreeDeleted.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
```

Comment:
Have you stripped some info from .png file? Why and how is described in our Wiki IIRC.